resources/fixtures/config/barcode.yml

```diff
@@ -1,3 +1,9 @@
 file-type: 'png'
 barcode-type: 'code39'
+
+# determines the class that will be used to automatically assign barcodes to products, will typically be the same as
+# the barcode-type
+barcode-generator: 'code39'
```

Comment:
I guess we could leave it blank, since if it's not set it defaults to the one that matches the type. I quite like it being more explicit though to be honest